pkgs/development/compilers/llvm/common/openmp/default.nix

```
@@ -53,6 +59,8 @@ stdenv.mkDerivation (rec {
 
   buildInputs = [
     (if stdenv.buildPlatform == stdenv.hostPlatform then llvm else targetLlvm)
+  ] ++ lib.optionals (ompdSupport && ompdGdbSupport) [
```

```suggestion
  ] ++ lib.optionals ompdGdbSupport [
```

It's the gdb support specifically that needs Python, right? (I know it's the same thing but it's clearer to only check the specific thing.)

---

I feel like this makes it easier (if refactoring, for example) to lead to strange states where we're adding dependencies for features that were intended to be disabled.

The assert statement above does catch this, but I'd prefer not to rely on it.